Docs/README.md

```diff
@@ -8,7 +8,7 @@ https://www.itu.int/rec/T-REC-H.273
 
 A typical instructions for compiling on MacOS and Linux looks like this:
 1. cd into cd_inserter code directory
-2. clang++ -std=c++23 -arch arm64 -o cicp_inserter *.cpp
+2. `clang++ -std=c++23 -arch arm64 -o cicp_inserter *.cpp`
```

Comment:
We shouldn't fix this in this PR but I just noticed the '-arch arm64'. We should probably remove that.

Reply:
I agree (both parts)